core/src/test/java/apoc/trigger/TriggerNewProceduresTest.java

```diff
@@ -0,0 +1,583 @@
+package apoc.trigger;
+
+import apoc.nodes.Nodes;
+import apoc.util.TestUtil;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import org.neo4j.configuration.GraphDatabaseSettings;
+import org.neo4j.dbms.api.DatabaseManagementService;
+import org.neo4j.graphdb.GraphDatabaseService;
+import org.neo4j.graphdb.Node;
+import org.neo4j.graphdb.QueryExecutionException;
+import org.neo4j.graphdb.Relationship;
+import org.neo4j.graphdb.ResourceIterator;
+import org.neo4j.graphdb.Transaction;
+import org.neo4j.kernel.api.KernelTransaction;
+import org.neo4j.kernel.impl.coreapi.TransactionImpl;
+import org.neo4j.test.TestDatabaseManagementServiceBuilder;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+import java.util.concurrent.TimeUnit;
+
+import static apoc.ApocConfig.SUN_JAVA_COMMAND;
+import static apoc.trigger.TriggerNewProcedures.TRIGGER_BAD_TARGET_ERROR;
+import static apoc.trigger.TriggerNewProcedures.TRIGGER_NOT_ROUTED_ERROR;
+import static apoc.trigger.TriggerTestUtil.TIMEOUT;
+import static apoc.trigger.TriggerTestUtil.TRIGGER_DEFAULT_REFRESH;
+import static apoc.trigger.TriggerTestUtil.awaitTriggerDiscovered;
+import static apoc.util.TestUtil.*;
+import static org.junit.Assert.*;
+import static org.junit.jupiter.api.Assertions.fail;
+import static org.neo4j.configuration.GraphDatabaseSettings.procedure_unrestricted;
+import static org.neo4j.internal.helpers.collection.MapUtil.map;
+import static org.neo4j.test.assertion.Assert.assertEventually;
+
+/**
+ * Test class for non-deprecated procedures, 
+ * i.e. `apoc.trigger.install`, `apoc.trigger.drop`, `apoc.trigger.dropAll`, `apoc.trigger.stop`, and `apoc.trigger.start`
+ */
```

Comment:
Not sure how needed this comment is either, but if you'd like to keep this then could we perhaps also list the procedures apoc.trigger.show and apoc.trigger.list?